go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/go-errors/errors v1.0.1
 	github.com/go-openapi/strfmt v0.19.5 // indirect
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
```

Comment:
While we're touching all those proto files, should we also fix the error ` Missing 'go_package' option in "chainrpc/chainnotifier.proto"`?

Reply:
done